rdtools/normalization.py

```diff
@@ -95,11 +96,39 @@ def normalize_with_pvwatts(energy, pvwatts_kws):
         freq = energy.index.freq
 
     dc_power = pvwatts_dc_power(**pvwatts_kws)
+    irrad = pvwatts_kws['poa_global']
+
+    # Length of each right labeled interval
+    model_tds = (dc_power.index - dc_power.index.shift(-1)).astype('int64') / (10.0**9 * 3600.0)
+    irrad_tds = (irrad.index - irrad.index.shift(-1)).astype('int64') / (10.0**9 * 3600.0)
+    measure_tds = (energy.index - energy.index.shift(-1)).astype('int64') / (10.0**9 * 3600.0)
```

Comment:
If I am reading this right, when you use `.astype('int64')` on a timedelta object, the result is in nanoseconds?